ui/plugin-system/src/runtime/variables.ts

```diff
@@ -124,22 +124,22 @@ export class VariableStoreStateMap {
   }
 }
 
-export type TemplateVariableSrv = {
+export type VariableSrv = {
   state: VariableStateMap;
 };
 
-export const TemplateVariableContext = createContext<TemplateVariableSrv | undefined>(undefined);
+export const VariableContext = createContext<VariableSrv | undefined>(undefined);
 
-function useTemplateVariableContext() {
-  const ctx = useContext(TemplateVariableContext);
+function useVariableContext() {
+  const ctx = useContext(VariableContext);
   if (ctx === undefined) {
-    throw new Error('No TemplateVariableContext found. Did you forget a Provider?');
+    throw new Error('No VariableContext found. Did you forget a Provider?');
   }
   return ctx;
 }
 
-export function useTemplateVariableValues(names?: string[]): VariableStateMap {
-  const { state } = useTemplateVariableContext();
+export function useVariableValues(names?: string[]): VariableStateMap {
```

Comment:
useTemplateVariableValues was used only in that file so maybe this is better that you rename this one into something else or even merge the two functions what do you think ?

Reply:
This is the same for the useTemplateVariableValues inside the TemplateVariableProvider.tsx

![image](https://github.com/perses/perses/assets/10258608/8ff15f90-5242-44a2-926a-ca70e15eacc6)

Reply:
2nd PR coming for this ;) I really wanted to focus only on the "Template" word removal since there are a lot of files modified